app/src/main/res/layout/fragment_server.xml

```diff
@@ -49,7 +49,6 @@
         app:layout_constraintTop_toBottomOf="@+id/title"
         app:rowHeight="153dp"
         tools:itemCount="4"
-        tools:layout_height="120dp"
```

Comment:
I guess this change was also accidental?

Reply:
Well yes and no. I included it as imo it's a change not worthy of a PR as it's dev-facing only. 

Removing that lines allows the preview to actually reflect the final UI and I have no idea why that value was there to begin with.

Reply:
You're right, just checked and removing this line does actually improve the preview.